tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/ompl/ompl_planner_configurator.h

```diff
@@ -36,10 +36,29 @@
 TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <ompl/base/SpaceInformation.h>
 #include <ompl/base/Planner.h>
+#include <tinyxml2.h>
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 namespace tesseract_planning
 {
+enum struct OMPLPlannerType : int
```

Comment:
I know they are the same but for consistency with other enums in the code base please switch to `enum class`.